setup.cfg

```diff
@@ -1,7 +1,13 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [bdist_wheel]
 universal = 1
 
 [flake8]
+doctests = 1
```

Comment:
Oh, I've only just now realized the additional flake8 errors I had to additionally fix came from `doctests = 1` here!